bgpd/bgp_vty.c

```diff
@@ -5632,6 +5647,18 @@ DEFUN (address_family_encapv6,
   return CMD_SUCCESS;
 }
 
+DEFUN (address_family_evpn,
```

Comment:
If evpn not compiled, you will see the node, but you will not be able to enter in sublevel.

Maybe you think of not installing at all evpn nodes at all ?

the ifdef is only for prefix access, currently